src/Forms/Settings.cs

```diff
@@ -2553,5 +2558,18 @@ private void linkLabelOpenDictionaryFolder_LinkClicked(object sender, LinkLabelL
             System.Diagnostics.Process.Start(dictionaryFolder);
         }
 
+        private void textBoxVlcPath_MouseLeave(object sender, EventArgs e)
+        {
+            try
+            {
+                var path = textBoxVlcPath.Text.Trim('\"');
+                if (path.Length > 3 && Path.IsPathRooted(path) && Path.GetFileName(path).Equals("vlc.exe", StringComparison.OrdinalIgnoreCase) && File.Exists(path))
+                {

```

Comment:
The .exe extension is an MS Windows peculiarity. This test might fail needlessly on a different OS.